src/main/resources/db/migration/all/20240311103635__make_ap_type_column_on_approved_premises_applications_table.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE approved_premises_applications ADD COLUMN ap_type TEXT NOT NULL DEFAULT 'NORMAL';
```

Comment:
it may be worth splitting this into two migrations. one to add the column and one to do the backfill. Otherwise if backfill fails for some reason (not that it should) we don't have to manually remove the column to re-run this migration once fixed?

Reply:
I've split this into multiple migrations (including one for the `DROP`ping of columns).